src/main/webapp/sass/_core.scss

```diff
@@ -3431,4 +3431,12 @@ div.border {
 
 #body .EntityFinderSelectionPane.alert.alert-warning {
     border-left: none;
+}
+
+.entityFinder .synAlertPanel {
+    display: flex;
+    > * {
+        width: 600px;
+        flex-grow: 1;
+    }
 }
```

Comment:
Kinda hacky, lets us fill the modal width without going over (can't use `width: 100%` because parent has `width: max-content`)